src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -560,6 +561,9 @@ private static class Log
             private static readonly Action<ILogger, Exception> _abortFailed =
                 LoggerMessage.Define(LogLevel.Trace, new EventId(8, "AbortFailed"), "Abort callback failed.");
 
+            private static readonly Action<ILogger, Exception> _clientTimeout =
+                LoggerMessage.Define(LogLevel.Debug, new EventId(9, "ClientTimeout"), "Client has not sent a message in too long. Closing connection.");
```

Comment:
Maybe "Client has been inactive for too long"
or "Client hasn't sent a message in the Timeout period". I kind of want to reference the timeout but it's not a big deal